packages/website/docs/RadioSelectConnected.md

```diff
@@ -38,14 +38,14 @@ Always preselect an option as the default. The default option can identify:
 -   The most commonly selected option when you want to help expedite the task.
     If preselecting a default option increases the risk of irreversible changes or security issues, always use the least risky option as the default.
 
-When the user needs to be able to easily revert to the default option, for instance when testing configurations, add “(recommended)” or “(default)” to the appropriate option label.
+When the user must easily revert to the default option, for instance when testing configurations, add “(recommended)” or “(default)” to the appropriate option label.
```

Comment:
Again, not sure the meaning is 100% the same

Reply:
okay, this one I'll revert